CONTRIBUTING.md

```diff
@@ -251,6 +251,7 @@ should be targeted against the release candidate branch.
   and ensure it's protected against pushing from anyone except the release
   manager/coordinator
     - **no PRs targeting this branch should be merged unless exceptional circumstances arise**
+- Bump defaultCosmosSDKVersion in `version/version.go`
```

Comment:
Wait, we don't need do this anymore, right?

Reply:
yup will fix, thanks